code/modules/mob/living/carbon/xenomorph/facehuggers.dm

```diff
@@ -234,27 +235,25 @@
 	update_icon()
 
 /obj/item/clothing/mask/facehugger/throw_impact(atom/hit_atom, speed)
-	if(stat == DEAD)
+	if(stat != CONSCIOUS)
```

Comment:
```suggestion
	if(stat)
```

Reply:
While technically true it's MUCH less readable.

Reply:
ultimately, this is why the CONSCIOUS define even exists.

Reply:
I prefer `if(stat)` myself, but this is not something we should be enforcing. `if(stat != CONSCIOUS)` is fine.